functions-app/calculate-score-v2/v1.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const moment = require('moment')
+
+const config = require('../config')
+const checkWindowDataService = require('./check-window.data.service')
+const schoolScoresDataService = require('./schools-scores.data.service')
+const schoolDataService = require('./school.data.service')
+const pupilResultsDiagnosticCache = require('./pupil-results-diagnostic-cache.data.service')
+const redisCacheService = require('../lib/redis-cache.service')
+
+let logger
+
+const v1 = {
+  process: async function (context) {
+    logger = context.log
+    redisCacheService.setLogger(logger)
+    await handleStoreSchoolsScores(context)
```

Comment:
that method name is a tongue twister.  can you rename to something like calculateSchoolScores please

Reply:
@GuyHarwood changed to `calculateSchoolResults`